modules/monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling the platform Alertmanager instance for user-defined projects
+
+You can enable user-defined alerts to use the main Alertmanager instance to send notifications for user-defined projects. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have enabled monitoring for user-defined projects in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
```

Comment:
`You have enabled monitoring for user-defined projects in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.`
should be removed

Reply:
Removed